deepchecks/vision/utils/accuracy_multiclass.py

```diff
@@ -0,0 +1,41 @@
+from ignite.metrics import Metric
```

Comment:
File should be removed, or otherwise changed to single-class metric used in example notebook of custom metric.